lib/helper/Polly.js

```diff
@@ -92,6 +100,15 @@ class Polly extends Helper {
     });
   }
 
+  // Connect WebDriver helper to mock future requests.
+  async _connectWebDriver(title) {
```

Comment:
Requires a simple docblock, as this method can be executed from other helpers. And it will be available in documentation...